pantheon-systems/WordPress
# Wordpress 4.4

For more information, please see: https://wordpress.org/news/2015/12/clifford/

Instructions to test:

- Create a new WordPress site on Pantheon.
- When site creation is finished, visit dashboard.
- Switch to "git" mode.
- Clone your site locally.
- Apply the files from this PR on top of your local checkout.
  - git remote add wp git@github.com:pantheon-systems/WordPress
  - git fetch wp
  - git merge wp/wordpress-4.4
- Push your files back up to Pantheon.
- Switch back to sftp mode.
- Visit your site. Step through the installation process.
- Confirm that the checksums of all of the core files match:
  - terminus --site=YOURWORDPRESSSITE wp core verify-checksums
  - Should say: "Success: WordPress install verifies against checksums."
